app/code/Magento/OfflineShipping/Model/ResourceModel/Carrier/Tablerate/CSV/ColumnResolver.php

```diff
@@ -12,7 +12,7 @@ class ColumnResolver
     const COLUMN_REGION = 'Region/State';
     const COLUMN_ZIP = 'Zip/Postal Code';
     const COLUMN_WEIGHT = 'Weight (and above)';
-    const COLUMN_WEIGHT_DESTINATION = 'Weight (and above)';
```

Comment:
Why did you change resource model?

Reply:
PHPStan reported that values are overriden on lines 25 - 26. I verified implementation and noticed that author of the test actually used coincidence to make his tests pass. :-)

Reply:
How can affect the functionality? Should it be covered by test?

Reply:
We can just remove the test that refers to these `const`s :-) or create `private const` inside the test with correct consts, and leave resource model still wrong :-)

Reply:
@lbajsarowicz In order to deliver updated tests as easy as possible our intention is not to change anything except for tests. That will allow us to run only Unit and Static tests and avoid additional load for regression. You can create a separate PR that fixes the issue

Reply:
Ok. I'll put the phpstan ignore on that error then and rollback  fix

Reply:
But please create a separate fix just not to lose it :)